corehq/apps/hqwebapp/templates/hqwebapp/partials/get_demo_banner.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div class="alert alert-warning alert-trial">
+  <i class="fa fa-info-circle"></i>
+  {% url 'domain_subscription_view' domain as url_sub %}
+  {% with num_trial_days_remaining|pluralize as days_plural %}
```

Comment:
I don't think this `days_plural` logic will necessarily pluralize correctly for translations (`día` => `dias`).